resources/js/app-layout.js

```diff
@@ -78,6 +78,10 @@ window.ProcessMaker.navbar = new Vue({
         alertDismissed(alert) {
             const index = this.alerts.indexOf(alert);
             index > -1 ? this.alerts.splice(index, 1) : null;
+            //remove old alerts
+            this.alerts = this.alerts.filter(item => {
+                return ((Date.now() - item.timestamp)/1000) < 5;
```

Comment:
@marcoAntonioNina is this necessary now that we have reduced the default time and only allow 5 messages?

If you think it is still is needed, can we make this alertValue instead of hard-coded '5', and if alertValue is 0, then do not automatically remove it? This section of code is overwriting the showValue that can be passed in

Reply:
@nolanpro you are right. I will modify to use the configured time.